metadata.json

```diff
@@ -153,28 +153,7 @@
       "metadata": {
         "pipelinePlatform": "node"
       },
-      "versions": [
-        {
-          "id": "v8-community",
-          "name": "8.x (Community)"
-        },
-        {
-          "id": "v10-community",
```

Comment:
@helio-frota I think you removed too much here, right? You probably should only have removed the v8 ones?

Reply:
Thanks again. My mistake due distraction. Fixed :+1: